src/pages/Artist/Artist.js

```diff
@@ -1,15 +1,89 @@
 import React from 'react'
-import { useTranslation } from 'react-i18next'
-import { PageTitle } from '../../components'
 import { setPageTitle } from '../../utils/setPageTitle'
+import { useArtist } from '../../services/artists/query'
+import {
+  Container,
+  ImageContainer,
+  InfoContainer,
+  IconContainer,
+} from './Artist.styles'
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import {
+  faFacebook,
+  faSoundcloud,
+  faVimeoV,
+  faYoutube,
+  faInstagram,
+} from '@fortawesome/free-brands-svg-icons'
 
-export default function Artist() {
-  const { t } = useTranslation()
+library.add(faFacebook, faSoundcloud, faVimeoV, faYoutube, faInstagram)
+
+export default function Artist(artistSlug) {
+  const {
+    name,
+    description,
+    image_url,
+    fb_link,
+    sc_link,
+    vimeo_link,
+    yt_link,
+    ig_link,
+  } = useArtist(artistSlug)
 
   React.useEffect(() => {
-    setPageTitle(t('artists'))
+    setPageTitle(name)
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [])
 
-  return <PageTitle>{t('artists')}</PageTitle>
+  return (
+    <Container>
+      <ImageContainer image={image_url} />
+      <InfoContainer>
+        <div>
+          <p>{name}</p>
+          <IconContainer>
+            <a href={fb_link} target="_blank" rel="noreferrer">
+              <FontAwesomeIcon
+                icon={faFacebook}
+                style={{ marginRight: '.5em' }}
+                color="black"
+              />
+            </a>
+
+            <a href={yt_link} target="_blank" rel="noreferrer">
+              <FontAwesomeIcon
+                icon={faYoutube}
+                style={{ marginRight: '.5em' }}
+                color="black"
+              />
+            </a>
+            <a href={sc_link} target="_blank" rel="noreferrer">
+              <FontAwesomeIcon
+                icon={faSoundcloud}
+                style={{ marginRight: '.5em' }}
+                color="black"
+              />
+            </a>
+            <a href={vimeo_link} target="_blank" rel="noreferrer">
+              <FontAwesomeIcon
+                icon={faVimeoV}
+                style={{ marginRight: '.5em' }}
```

Comment:
this better to use styled component 
1. Create a styled-component `<ArtistIcon />` that contains margins colors etc...
2. Assign `FontAwesomeIcon` AS root element  `<ArtistIcon as={FontAwesomeIcon} />`
3. Add complementary props `<ArtistIcon as={FontAwesomeIcon} icon={faVimeoV} />`

Reply:
je connaissais pas le as={} bienvu !!!!